docs/release-playbook.md

```diff
@@ -0,0 +1,205 @@
+# Bazel Release Playbook
+
+Authors: lpino@, pcloudy@, laurentlb@
+
+Status: Work in progress
+
+Last Updated: 2018-11-09
+
+Link to this document: go/bazel-release-playbook
+
+Congratulations, you're the new release manager!
```

Comment:
Rephrase: the page should be relevant for anyone who wants to know how the release process works

Reply:
Done.